eclair-core/src/main/scala/fr/acinq/eclair/router/Graph.scala

```diff
@@ -175,7 +175,6 @@ object Graph {
    * @param wr                 ratios used to 'weight' edges when searching for the shortest path
    */
   private def dijkstraShortestPath(g: DirectedGraph,
-                                   sender: PublicKey,
```

Comment:
Clever, you've been able to remove the need for that field by reversing the flow of k-shortest path, I like it!